src/content/docs/tutorial-create-alerts/create-new-relic-alerts.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: "Creating New Relic alerts"
```

Comment:
I think the titles of all docs in this series (and the nav titles) need to be much more clearly distinguished from the standard alerts docs. Perhaps even using`tutorial` in the title.